pandas/tools/plotting.py

```diff
@@ -1554,8 +1554,11 @@ class PlanePlot(MPLPlot):
 
     _layout_type = 'single'
 
-    def __init__(self, data, x, y, **kwargs):
-        MPLPlot.__init__(self, data, **kwargs)
+    def __init__(self, data, x, y, sharex=False, **kwargs):
+        if sharex is None:
+            # Fix x axis color bar problems
+            sharex = False
```

Comment:
This sharex fix, is this independent from the datetime fix? 
If so, can you make a separate PR for it?